domain/src/main/kotlin/com/yapp/bol/validate/NicknameValidator.kt

```diff
@@ -1,15 +1,19 @@
 package com.yapp.bol.validate
 
-import com.yapp.bol.InvalidNicknameException
-
 object NicknameValidator {
     private const val MIN_NICKNAME_LENGTH = 1
     private const val MAX_NICKNAME_LENGTH = 10
+    private val regex = Regex("^[a-zA-Z0-9_가-힣]*$") // 한글, 영어, 숫자, 언더 스코어
+
+    fun validate(nickname: String): Boolean {

```

Comment:
```suggestion
    fun isValidate(nickname: String): Boolean {

```

p2. Boolean을 리턴 할거면 위 네이밍이 좋아 보입니다.
실제로 그런 실수가 있어서 (샌박 배포를 위해) 수정 해 뒀습니다.

validate 는 assert~ 처럼 자체적으로 Exception을 throw 할 것 처럼 보여요